src/tools/PrizeTierController/ProjectionView/PrizePoolItem.tsx

```diff
@@ -0,0 +1,409 @@
+  const { data: tvl, isFetched: isFetchedTvl } = usePrizePoolTvl(prizePool)
+  const {
+    register,
+    watch,
+    setValue,
+    formState: { errors }
+  } = useForm<ProjectionSettings>({
+    mode: 'onChange',
+    defaultValues: defaultFormValues,
+    shouldUnregister: true
+  })
+  const { t } = useTranslation()
+
+  const formTvl = watch('tvl')
+  const parsedFormTvl = parseFloat(formTvl)
+  const formVariance = watch('variance')
+  const parsedFormVariance = parseFloat(formVariance)
+
+  useEffect(() => {
+    if (isFetchedTvl) {
+      if (formTvl === '0' || formTvl === undefined) {
+        setValue('tvl', tvl.toFixed(0))
+      }
+    }
+  }, [tvl, isFetchedTvl, formTvl, parsedFormTvl])
+
+  const dprMultiplier = !!parsedFormTvl
+    ? calculateDprMultiplier(
+        prizeTier.dpr,
+        parsedFormTvl,
+        prizeTier.prize,
+        prizeTierHistoryContract.token.decimals
+      )
+    : 0
+
+  const varianceMultiplier =
+    !!parsedFormVariance && parsedFormVariance >= -100 && parsedFormVariance <= 100
+      ? 1 + parsedFormVariance / 100
+      : 1
+
+  const prizes = prizeTier.tiers.map((tier, i) =>
+    parseFloat(
+      formatUnits(
+        calculate.calculatePrizeForTierPercentage(i, tier, prizeTier.bitRangeSize, prizeTier.prize),
+        parseInt(prizeTierHistoryContract.token.decimals)
+      )
+    )
+  )
+
+  const numPrizesPerTier = calculate.calculateNumberOfPrizesPerTier(prizeTier)
+  const prizeChances = numPrizesPerTier.map((value) => value * dprMultiplier * varianceMultiplier)
+  const expectedNumPrizes = prizeChances.reduce((a, b) => a + b, 0)
+  const expectedTierPrizeAmounts = prizes.map((prize, i) => prize * prizeChances[i])
+  const expectedPrizeAmount = expectedTierPrizeAmounts.reduce((a, b) => a + b, 0)
+
+  return (
+    <>
+      {isFetchedTvl ? (
+        <>
+          <BasicStats
+            tvl={parsedFormTvl}
+            dpr={prizeTier.dpr}
+            token={prizeTierHistoryContract.token}
+            className='mb-2'
+          />
+          <PrizesOverTime
+            numPrizes={expectedNumPrizes}
+            prizeAmount={expectedPrizeAmount}
+            className='mb-2'
+          />
+          <DrawBreakdown prizes={prizes} prizeChances={prizeChances} className='mb-2' />
+          <AdvancedOptions
+            tvl={tvl}
+            errors={errors}
+            register={register}
+            watch={watch}
+            setValue={setValue}
+          />
+        </>
+      ) : (
+        t('loading')
+      )}
+    </>
+  )
+}
+
+const BasicStats = (props: { tvl: number; dpr: number; token: Token; className?: string }) => {
+  const { tvl, dpr, token, className } = props
+  const { t } = useTranslation()
+
+  return (
+    <div className={classNames('flex flex-col', className)}>
+      <SectionTitle text={t('basicStats')} />
+      <span>
+        TVL: {tvl.toLocaleString('en', { maximumFractionDigits: 0 })} {token.symbol}
```

Comment:
Use `formatNumberForDisplay` rather than the lower level call, we've got some wrapper code that fixes a very sneaky bug In what you have written here and it'll be easier to migrate to a new function once we incorporate changing locales/currencies in hte future.

Reply:
Fixed through 0863a8a